File: OCH_Tao/ABC/A/067.py
A,B = map(int,input().split())
if A % 3 == 0:
  print("Possible")
elif B % 3 == 0:
  print("Possible")
elif (A+B) % 3 == 0:
  print("Possible")
else:
  print("Impossible")

File: OCH_Tao/ABC/A/067.js
const AB = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
if (AB[0] % 3 === 0) {
  console.log("Possible");
} else if (AB[1] % 3 === 0) {
  console.log("Possible");
} else if ((AB[0] + AB[1]) % 3 === 0) {
  console.log("Possible");
} else {
  console.log("Impossible");
}